xdsl/dialects/onnx.py

```diff
@@ -26,6 +26,100 @@
 from xdsl.utils.exceptions import VerifyException
 
 
+def verify_unidirectional_broadcast_shape(
+    lhs: TensorType[Attribute], rhs: TensorType[Attribute], res: TensorType[Attribute]
+) -> None:
+    """
+    Returns a unidirectional broadcastable shape
+    """
+    lhs_shape = lhs.get_shape()
+    rhs_shape = rhs.get_shape()
+    expected_shape = unidirectional_broadcast_shape(list(lhs_shape), list(rhs_shape))
+    if not expected_shape:
```

Comment:
```suggestion
    if expected_shape is not None:
```

Reply:
you mean is `None` right? because that would fail tests?